src/System.Security.Cryptography.Pkcs/pkg/System.Security.Cryptography.Pkcs.pkgproj

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Pkcs.csproj">
+      <SupportedFramework>net46;net461;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
This list is driving the validation tests that happen during packaging.  It's saying that for each of this frameworks (and their relevant RIDs) that the package should be supported at the API level exposed by the reference assembly.

Listing both net46 and net461 is redundant.  The rest looks OK.